cortex-js/src/infrastructure/commanders/models/model-start.command.ts

```diff
@@ -44,6 +44,7 @@ export class ModelStartCommand extends CommandRunner {
 
   async run(passedParams: string[], options: ModelStartOptions): Promise<void> {
     let modelId = passedParams[0];
+    console.log('Finding model...');
```

Comment:
I removed a similar log. It’s super fast, better log Loading model… (I added this)